### PR TITLE
Feature/forgot password

### PR DESCRIPTION
**Pull request by mjvkeulen**

### CODE REVIEW

File: src/vue/src/router/index.js [hunk 1]
```diff
@@ -33,9 +34,14 @@ var router = new Router({
         name: 'Home',
         component: Home
     }, {
-        path: '/Login',
+        path: '/Login/',
```
Comment (Larspolo) [inline src/vue/src/router/index.js]: Why the /? Now its inconsistent with /Home right?

File: src/django/test/test_get_apis.py [hunk 2]
```diff
@@ -19,19 +19,20 @@ class GetApiTests(TestCase):
...
-        self.username, self.password, self.user = test.set_up_user_and_auth('test', 'test123')
-        self.rein_user, self.rein_pass, self.rein = test.set_up_user_and_auth("Rein", "123")
-        self.no_perm_user, self.no_perm_pass, self.no_permission_user = test.set_up_user_and_auth("no_perm", "123")
+        self.username, self.password, self.user = test.set_up_user_and_auth('test', 'test123', 'testtt@te.com')
+        self.rein_user, self.rein_pass, self.rein = test.set_up_user_and_auth("Rein", "123", 're@rein.com')
+        self.no_perm_user, self.no_perm_pass, self.no_permission_user = test.set_up_user_and_auth("no_perm", "123",
+                                                                                                  'sigh@sigh.com')
         self.course = factory.make_course("Beeldbewerken", "BB")
-        self.lars = factory.make_user("Lars", "123")
+        self.lars = factory.make_user("Lars", "123", "l@l.com")
```
[the remark this answers is not in the document]
Comment (mjvkeulen) [reply src/django/test/test_get_apis.py]: Haha simply first letter of username @ username . com

File: src/vue/src/router/index.js (hunk 1 above)
Comment (mjvkeulen) [reply src/vue/src/router/index.js]: Agreed updated

File: src/django/VLE/views/update.py [hunk 3]
```diff
@@ -794,3 +796,84 @@ def update_user_profile_picture(request):
...
+        if request.data['new_password'] == request.data['new_password'].lower():
+            return responses.bad_request('Password needs to contain at least 1 capital letter.')
+        if re.match(r'^\w+$', request.data['new_password']):
+            return responses.bad_request('Password needs to contain a special character.')
+
+        user.set_password(request.data['new_password'])
+        user.save()
+        return responses.success(message='Succesfully changed the password, please login.')
+    else:
+        return responses.bad_request('Invalid recovery token.')
```
[the remark this answers is not in the document]
Comment (mjvkeulen) [reply src/django/VLE/views/update.py]: Hm, true but I quite like the explicit else. Is this not simply a matter of style?

[the remark this answers is not in the document]
Comment (Larspolo) [reply src/django/VLE/views/update.py]: In forgot password you have a `if not user: return...` for example

**Pull request merged**